rotkehlchen/tests/api/active_management/test_transfers.py

```diff
@@ -119,6 +126,13 @@ def test_transfers(
 
 @pytest.mark.vcr(filter_query_parameters=['apikey'])
 def test_get_token_balance(rotkehlchen_api_server: APIServer) -> None:
+    rotkehlchen_api_server.rest_api.rotkehlchen.set_settings(
+        settings=ModifiableDBSettings(evm_indexers_order=SerializableChainIndexerOrder(
+            order={chain_id: [EvmIndexer.ETHERSCAN] for chain_id in [ChainID.BASE, ChainID.BINANCE_SC, ChainID.OPTIMISM]},  # noqa: E501
+        )),
+    )
+    # TODO: figure out why the db_settings fixture didn't have any effect when setting the indexer
```

Comment:
I think it's for same reason as I did in some base tests recently. The fixture is not respected for that setting